docs/elements/pl-string-input.md

```diff
@@ -34,6 +34,7 @@ def generate(data):
 | `ignore-case`             | boolean                 | false           | Whether to enforce case sensitivity (e.g. "hello" != "HELLO").                                                                                                            |
 | `normalize-to-ascii`      | boolean                 | false           | Whether non-English characters (accents, non-latin alphabets, fancy quotes) should be normalized to equivalent English characters before submitting the file for grading. |
 | `placeholder`             | string                  | —               | Hint displayed inside the input box describing the expected type of input.                                                                                                |
+| `initial-value`           | string                  | —               | Initial value is added to the text box the first time it is rendered.                                                                                                     |
```

Comment:
We also need an example in `exampleCourse/questions/element/stringInput/question.html`.

Reply:
Done in [53578e3](https://github.com/PrairieLearn/PrairieLearn/pull/13494/commits/53578e3417cbe264a74f0843b67ab8dd464d5e74)